pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -164,7 +164,7 @@ class DialogFooter extends React.Component {
                 className="apply"
                 variant={ action.style || "secondary" }
                 onClick={ this.action_click.bind(this, action.clicked) }
-                isDisabled={ actions_disabled || ('disabled' in action && action.disabled) }
+                isDisabled={ actions_disabled || ('disabled' in action?.disabled) }
```

Comment:
This is wrong now, you meant just `|| action?.disabled }`

Reply:
After revising operator precedence in JavaScript, I realize how wrong this was.